subprojects/docs/src/samples/maven-publish/javaProject/build.gradle

```diff
@@ -27,14 +27,28 @@ publishing {
             }
         }
     }
+}
 // END SNIPPET publish-custom-artifact
+
+// START SNIPPET repo-url-from-variable
+ext {
```

Comment:
Why put this in the ext namespace? A local variable in the publishing block would be simpler.